app/src/gui/layout/appBar.tsx

```diff
@@ -197,45 +251,64 @@ export default function MainAppBar(props: NavbarProps) {
     },
     projectList === null
       ? {
-          title: 'Loading notebooks...',
-          icon: <AccountTree />,
-          to: '/',
-          disabled: true,
-        }
+        title: 'Loading notebooks...',
+        icon: <DescriptionIcon />,
+        to: '/',
+        disabled: true,
+      }
       : isAuthenticated
         ? getNestedProjects(projectList)
         : {
-            title: 'Notebooks',
-            icon: <AccountTree />,
-            to: '/',
-            disabled: true,
-          },
-  ];
-  const bottomMenuItems: Array<MenuItemProps> = [
-    isAuthenticated
-      ? {
-          title: 'User',
-          icon: <AccountCircleIcon />,
-          to: ROUTES.SIGN_IN,
-          disabled: false,
-        }
-      : {
-          title: 'User',
-          icon: <AccountCircleIcon />,
+          title: 'Notebooks',
+          icon: <DescriptionIcon />,
           to: '/',
           disabled: true,
         },
+  ];
+
+  const bottomMenuItems: Array<MenuItemProps> = [
+
+    // @TODO Ranisa: Commented this for now, to be discussed if needed to display this nav item
+    // {
+    //   title: 'About Build',
+    //   icon: <SettingsIcon />,
+    //   to: ROUTES.ABOUT_BUILD,
+    //   disabled: false,
+    // },
+
     {
-      title: 'About Build',
-      icon: <SettingsIcon />,
-      to: ROUTES.ABOUT_BUILD,
+      title: 'Help',
+      icon: <HelpIcon />,
+      to: ROUTES.HELP,
       disabled: false,
     },
+
+    {
+      title: 'Switch org',
+      icon: <SwapHorizIcon />,
+      to: ROUTES.SWITCH_ORG,
+      disabled: false,
+    },
```

Comment:
I don't think we should add this until we have this functionality. Confusing for users if it does nothing.

Reply:
Got your point. Removing this and help.